README.md

```diff
@@ -1464,6 +1465,16 @@ Type | Interface               | Value type | Description
 out  | evt.notification.report | object     | Notification reports that are sent by devices.
 out  | evt.sensor.report       | object     | Multilevel sensor reports with unknown sensor type or/and unknown unit.
 out  | evt.meter.report        | object     | Meter reports with unknown meter type or/and unit.
+in   | evt.meter.reset         | null       | Resets all historical readings.
+
+#### Service props
+
+Name               | Value example          | Description
+-------------------|------------------------|-------------
+`meter_type`       | "meter_elec", "6"      | Meter type - described as text when known and number if unknown.
```

Comment:
Not sure if I understand the purpose of these properties. This service can be utilized to channel events/readings from multiple meters, so why it has single meter_type property?

The same goes for supported units, export units or rate types. If you need to put it here, then it should be a complex map of supported units per particular meter type. But do we really need this information? We will have it post factum and since we don't implement interface for active query commands, how should we use this information?

Reply:
From what I understand services are created per endpoint and since specific endpoints return only one meter type I didn't see a reason to allow multiple meter types here. 
I don't know yet what kind of information is useful for the interface (so that it can show something or expect some specific info before it receives first report from the device), but if you say it's not useful at all then I'll remove these properties :smiley: